docs/content/reference/migration/migration-0-17.md

```diff
@@ -41,7 +41,7 @@ For example, setting color and enabling the `SeriesPoint` visualizer was previou
 rr.log("data", rr.SeriesPoint(color=[255, 255, 0]), static=True)
 
 for t in range(1000):
-    rr.set_index("frame_nr", sequence=t)
+    rr.set_time("frame_nr", sequence=t)
```

Comment:
Probably search-and-replace error, but we should not change old migration guides.

Reply:
Same here - this is reverting a search-and-replace error in an earlier PR 🤦

Reply:
Good catch!